src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  Collapse,
+  Grid,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import lostAndFoundService from 'services/lostAndFound';
+import styles from '../../../../views/CampusSafety/views/LostAndFound/LostAndFound.module.css'; // Import the external CSS
+import { MissingItemReport } from 'services/lostAndFound'; // Import the type from the service
+import { DateTime } from 'luxon';
+import { useWindowSize } from 'hooks';
+
+const formatDate = (date: string) => {
+  return DateTime.fromISO(date).toFormat('MM-dd-yyyy'); // Adjust format as needed
+};
+
+const noReports = (
+  <Grid item alignItems="center">
+    <br />
+    <Typography variant="h5" align="center">
+      No Missing Items Reported Yet
+    </Typography>
+    <br />
+  </Grid>
+);
+
+const LostAndFoundCard = () => {
+  const [activeReports, setActiveReports] = useState<MissingItemReport[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [width] = useWindowSize();
+
+  useEffect(() => {
+    const fetchMissingItems = async () => {
+      try {
+        setLoading(true);
+        const reports: MissingItemReport[] = await lostAndFoundService.getMissingItemReportUser();
+
+        // Map the reports into active and past reports
+        const active = reports
+          .filter((report) => report.status === 'active') // Filter for non-found items
+          .sort((a, b) => new Date(b.dateCreated).getTime() - new Date(a.dateCreated).getTime())
+          .map((report) => ({
+            ...report,
+            colors: report.colors || [], // Ensure colors is an array
+          }));
+
+        setActiveReports(active);
+      } catch (error) {
+        console.error('Error fetching missing items:', error);
+      } finally {
+        setLoading(false);
+      }
+    };
+    fetchMissingItems();
+  });
+
+  const renderReportContent = (report: MissingItemReport) => (
+    <>
+      <Grid item xs={width >= 900 ? 4 : 12}>
+        <Typography align="center">{formatDate(report.dateLost)}</Typography>
+      </Grid>
+      <Grid item xs={width >= 900 ? 4 : 12}>
+        <Typography
+          align="center"
+          style={{ cursor: report.locationLost.length > 15 ? 'pointer' : 'default' }}
+        >
+          {report.locationLost.length <= 15
+            ? report.locationLost
+            : `${report.locationLost.slice(0, 15)}...`}
```

Comment:
There's a way to do this with pure CSS that's a lot cleaner and more adaptive than the JS method used here.  It's a bit annoying to get working, but it works a lot better because it dynamically adjusts the cutoff point based on the width of the screen.
@Centerville1 can work on that this week.